Tasks/DeployVisualStudioTestAgent/TestAgentConfiguration.ps1

```diff
@@ -599,7 +599,7 @@ function InvokeTestAgentConfigExe([string[]] $Arguments, [string] $Version, [Sys
         return $out
     }
 
-    throw "Did not find TestAgentConfig.exe at : $exePath. Ensure that TestAgent is installed."
+    throw (Get-LocalizedString -Key "Did not find TestAgentConfig.exe at : '{0}'. Ensure that TestAgent is installed." -ArgumentList $exePath)
```

Comment:
are these string in resx? also this is not directly run by task. but on test agent